compiler+runtime/src/cpp/main.cpp

```diff
@@ -141,6 +144,12 @@ namespace jank
     le.setPrompt(get_prompt("=> "));
     native_transient_string input{};
 
+    /* We write every REPL expression to a temporary file, which then allows us
+     * to later review that for error reporting. We automatically clean it up
+     * and we reuse the same file over and over. */
+    auto const tmp{ boost::filesystem::temp_directory_path() };
+    auto const path{ tmp / boost::filesystem::unique_path("jank-repl-%%%%.jank") };
```

Comment:
Thank you!

Reply:
Was this bothering you somehow? :grin:

Reply:
I found this file littered around my directories at some point.

Reply:
Hmmm. We weren't writing to a file before. Must've been from something else.

Reply:
Oh, probably `.jank-repl-history`.

Reply:
Ah, yes. You'll see that. We don't clean them up. Leiningen does the same with `.lein-repl-history`.

Reply:
Never been bothered by Leiningen's behavior here. Looks like it uses the project-root if project.clj is present, otherwise uses a config directory

https://github.com/technomancy/leiningen/blob/24fb93936133bd7fc30c393c127e9e69bb5f2392/src/leiningen/repl.clj#L152-L153

Reply:
Yeah, I'd prefer if we did something like that, too. Not sure how we'd want to do it, though. We could just use jank's cache dir, which is in `~/.cache/jank` by default. That would be a global history for all projects.

Reply:
Yes maybe jank should take a path to save configuration, that can be set by e.g., lein-jank.